addons/pinniped/config-controller/controllers/constants.go

```diff
@@ -4,7 +4,12 @@
 package controllers
 
 const (
-	//TODO: I kinda don't like that we're copying a lot of these from addons/pkg/constants
+	CascadeControllerV1alpha1Name = "🐙 v1alpha1 pinniped cascade controller"
```

Comment:
i am inclined to start splitting out and grouping our consts in sets like this, rather than one big list.  not a big deal if yall want to just keep it one `const ( ... )` block.

Reply:
Sure, I'm fine splitting them into blocks.